File: beginner/javascript/2165.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var texto = lines.shift();
var qtsCaracteres = texto.length;

if (qtsCaracteres > 140) {
    console.log("MUTE");
} else {
    console.log("TWEET");
}

File: beginner/java/2165.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String texto = scanner.nextLine();
        int qtsCaracteres = texto.length();

        if (qtsCaracteres > 140) {
            System.out.println("MUTE");
        } else {
            System.out.println("TWEET");
        }
    }
}
